src/content/learn/separating-events-from-effects.md

```diff
@@ -993,12 +993,12 @@ export default function Timer() {
   return (
     <>
       <h1>
-        Counter: {count}
+        Sayaç: {count}
         <button onClick={() => setCount(0)}>Reset</button>
       </h1>
       <hr />
       <p>
-        Every second, increment by:
+        Her saniye artıyor:
```

Comment:
Burasi tam oturmamis. Dilimizin yapisindan oturu direkt ceviremeyiz burayi. Ya kodu degistirmemiz lazim ya da:

`Saniyedeki artis miktari:` 

gibi bir ceviri yapilabilir. 

Asagida baska bir yerde daha var sanirim. Orayi da guncelleyin lutfen.

Reply:
Hataları uyardığınız için teşekkür ederim. Yeni PR'da düzeltildi.